src/library_glfw.js

```diff
@@ -1020,48 +1006,37 @@ var LibraryGLFW = {
         Browser.setCanvasSize(width, height);
       }
 
-      // Create context when there are no existing alive windows
-      for (i = 0; i < GLFW.windows.length && GLFW.windows[i] == null; i++) {
-        // no-op
-      }
-      var useWebGL = GLFW.hints[0x00022001] > 0; // Use WebGL when we are told to based on GLFW_CLIENT_API
-      if (i == GLFW.windows.length) {
-        if (useWebGL) {
-          var contextAttributes = {
-            antialias: (GLFW.hints[0x0002100D] > 1), // GLFW_SAMPLES
-            depth: (GLFW.hints[0x00021005] > 0),     // GLFW_DEPTH_BITS
-            stencil: (GLFW.hints[0x00021006] > 0),   // GLFW_STENCIL_BITS
-            alpha: (GLFW.hints[0x00021004] > 0)      // GLFW_ALPHA_BITS
-          }
+      // Use WebGL when we are told to based on GLFW_CLIENT_API
+      var useWebGL = GLFW.hints[0x00022001] > 0;
+      if (useWebGL && !Module.ctx) {
```

Comment:
This check for ctx seems like new behavior, but I might be reading the code wrong?